daml.yaml

```diff
@@ -1,9 +1,9 @@
 ## Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
 ## SPDX-License-Identifier: Apache-2.0
 
-sdk-version: 1.4.0
+sdk-version: 1.16.0
 name: dabl-integration-exberry
-version: 1.0.0
+version: 0.8.0
```

Comment:
Per a comment in another PR, this should go to something like `1.0.1`: https://github.com/digital-asset/daml-dit-integration-exberry/pull/19/files#r712361453